include/LightGBM/config.h

```diff
@@ -1013,6 +1013,34 @@ struct Config {
 
   #pragma endregion
 
+  #pragma region Category Encoder Parameters
```

Comment:
Could you please add tests for that category encoders work with or document if they don't
- `Dataset.add_features_from()` function
- `Dataset.set_feature_name()` function
- `feature_fraction` param
- `feature_fraction_bynode` param
- `monotone_constraints`, `feature_contri`, `forcedsplits_filename`, `interaction_constraints` and other params where user should specify all features in order or something like that (feature number/name)